dbms/src/Flash/Mpp/MPPTask.h

```diff
@@ -113,10 +129,14 @@ class MPPTask : public std::enable_shared_from_this<MPPTask>
     void setErrString(const String & message);
 
 private:
+    // We must ensure this member variable is put at this place to be destructed at proper time
+    MPPTaskMonitorHelper mpp_task_monitor_helper;
+
     // To make sure dag_req is not destroyed before the mpp task ends.
     tipb::DAGRequest dag_req;
     mpp::TaskMeta meta;
     MPPTaskId id;
+    const LoggerPtr log;
```

Comment:
Why change this

Reply:
> Why change this

This change is affected by the previous aborted implement, I will move it back.